changelog/60572.fixed

```diff
@@ -0,0 +1 @@
+Use the right crypto libary for salt.utils.crypt.reinit_crypto
```

Comment:
Why are you renaming the `changelog/60529.fixed` file? Aslo, the filename should be the issue number you are fixing.

Reply:
because its a copy from master and I was told to by @sagetherage

Reply:
The PR is saying this is fixing #60215. The changelog should match that?